builder/ConfTargets.cmake

```diff
@@ -76,6 +76,14 @@ endif()
 
 option( MFX_ENABLE_KERNELS "Build with advanced media kernels support?" ON )
 
+if ( ${version} STRGREATER "1025" )
+  set ( MFX_MCTF_OPTION_ALLOWED ON )
+else()
+  set ( MFX_MCTF_OPTION_ALLOWED OFF )
```

Comment:
I suggest to avoid MCTF in the name. We may have other options which appear from 1.25. I.e. better to name in this way:
`set( API_1_25_OPTIONS ON)`

And please, move this set _before_ defining any options...